src/applications/travel-pay/components/AppointmentsTable.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CLAIM_STATUS = {
+  SAVED: 'Saved',
+  INCOMPLETE: 'Incomplete',
+  IN_PROCESS: 'In Process',
+  CLAIM_SUBMITTED: 'Claim Submitted',
+  MANUAL_REVIEW: 'In Manual Review',
+  ON_HOLD: 'On Hold',
+  APPEALED: 'Appealed',
+  // Closed as a bunch of variants.
+  // TBD if we need to be more specific.
+  CLOSED: 'Closed',
+};
+
+export default function AppointmentsTable() {
```

Comment:
We should maybe call this `TravelClaimsTable` or something similar

Reply:
good call. Removed the table as designs shifted, but kept the updated naming in newer components